src/vs/workbench/services/textfile/browser/browserTextFileService.ts

```diff
@@ -34,4 +38,14 @@ export class BrowserTextFileService extends AbstractTextFileService {
 	}
 }
 
+class BrowserEncodingOracle extends EncodingOracle {
+	async getPreferredWriteEncoding(): Promise<IResourceEncoding> {
+		return { encoding: 'utf8', hasBOM: false };
+	}
+
+	async getReadEncoding(): Promise<string> {
+		return 'utf8';
+	}
+}
```

Comment:
Temporary measure to keep web version work as is.

Reply:
@gyzerok shouldn't this also have an override for `getWriteEncoding`?